css/aside.css

```diff
@@ -47,32 +47,29 @@
   .aside {
     height: 100%;
     width: 0;
-    padding: 0;
     position: fixed;
     z-index: 1;
     top: 0;
```

Comment:
Define the top and left 0 in the root directory so it will inherit by its child component you don't have to define it again

Reply:
Done the Changes